src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const partner = (sex === 'f')
+    ? 'husband\'s'
+    : 'wife\'s';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
```

Comment:
let's put element contents on the new line, fix it everywhere
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
```